documentation/docs/welcome.md

```diff
@@ -15,29 +15,29 @@ keywords:
 
 # Welcome to INX-Dashboard
 
-INX-Dashboard is a node configuration dashboard for node owners.
-It provides an overview of your nodes health and allows you to manage peers and plugins.
-You can browse the nodes database with the built-in explorer and check the current tangle activity in the visualizer.
+INX-Dashboard provides a web GUI tool to help you manage your node. It provides an overview of the node's status and health and allows you to choose its peers, as well as the enabled plugins. It has a built-in explorer of all nodes on the network and visualizer for activity on the Tangle.
```

Comment:
```suggestion
The INX-Dashboard provides a web GUI tool to help you manage your node. It provides an overview of the node's status and health and allows you to choose its peers and the enabled plugins. It has a built-in explorer of all nodes on the network and a visualizer for activity on the Tangle.
```

Reply:
Since INX-Dashboard is a name, I don't think that "the" belongs there. I would rather keep "as well as" instead of "and" to indicate that peers and plugins are not related in any way.

Reply:
It's a dashboard plugin, so it's either the dashboard or the plugin.